### PR TITLE
Update v1.1 docs

### PR DESCRIPTION
**Pull request by shreyasnagare**
I have updated the website based on the latest changes in the schema. Have a look at the preview (whenever it's ready).

### CODE REVIEW

**Comment by shreyasnagare** [comment]
I'm not sure if this is intended or I missed something in the discussions but when I try to generate a new `Brick.ttl`, I get a lot of "`brick:$1 a brick:$1 .`" triples (100+). 

The ontology navigation tree relies heavily on the subclass relationships. The absence of triples like `brick:CO2 rdfs:subClassOf brick:Gas .` and `brick:Fuel_Oil rdfs:subClassOf brick:Oil .` makes it look untidy.

**Comment by gtfierro** [comment]
> I'm not sure if this is intended or I missed something in the discussions but when I try to generate a new Brick.ttl, I get a lot of "brick:$1 a brick:$1 ." triples (100+).

This is a consequence of OWL reasoning. These triples always should have been generated. You can remove them easily enough

> The ontology navigation tree relies heavily on the subclass relationships. The absence of triples like brick:CO2 rdfs:subClassOf brick:Gas . and brick:Fuel_Oil rdfs:subClassOf brick:Oil . makes it look untidy.

Are these not there anymore? It sounds like you are applying OWL reasoning to the ontology before constructing the tree. You should recover the structure you're looking for if you do not apply OWL reasoning

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright 2019 Shreyas Nagare
+Copyright 2020 Shreyas Nagare
```
Comment (gtfierro) [inline LICENSE]: You don't have copyright on this. Change it to `Brick Development Team` like the Brick license

**Comment by shreyasnagare** [comment]
> This is a consequence of OWL reasoning. These triples always should have been generated. You can remove them easily enough
Yeah, I can remove them. I was wondering why only 115 classes had that.

> Are these not there anymore? It sounds like you are applying OWL reasoning to the ontology before constructing the tree. You should recover the structure you're looking for if you do not apply OWL reasoning
I'm not applying OWL reasoning manually. I'll recheck if `generate_brick.py` does it before serialization. If it doesn't, I'll need a few pointers on how to bring back the subclass relationships.

**Comment by gtfierro** [comment]
`generate_brick.py` does not apply reasoning to the ontology and the subclass relationships should not have gone anywhere. Can you look into your parsing logic to see what's up?

**Comment by gtfierro** [comment]
@shreyasnagare it looks like there are a couple minor issues with substances and quantities, specifically. I'll fix those, but concentrate on Point + Equipment + Location, etc

**Comment by gtfierro** [comment]
@shreyasnagare issue should be fixed by https://github.com/BrickSchema/Brick/pull/101 when I merge it

**Comment by shreyasnagare** [comment]
> Thanks for the copyright change!

Don't thank me for that :P. 
It was always supposed to be `Brick Development Team`. It was just a GitHub generated LICENSE when it was one of my private repositories and then I never changed it.

> @shreyasnagare issue should be fixed by BrickSchema/Brick#101 when I merge it

Thanks for addressing the issue so quickly. The preview looks good to me. Let me know if it's good enough to merge in.

**Comment by gtfierro** [comment]
Waiting on @jbkoh's approval of https://github.com/BrickSchema/Brick/pull/101 and then we can merge that in.

In the meantime, you can take the regenerated `Brick.ttl` out of the PR's branch (https://github.com/BrickSchema/Brick/blob/d31d5f2badba176f10d20ea938ebe836539a0da6/Brick.ttl) and test!

**Comment by gtfierro** [comment]
The PR is merged now if you want to use `Brick.ttl` from master

**Comment by shreyasnagare** [comment]
This adds the complete version number on the ontology pages (suggested by @jbkoh). This won't add the information if it's redundant (for example, in the v1.0.3 page).

Screen Shot 2020-01-30 at 6 26 04 PM

**Pull request merged**